src/frameworks/CoreServices/reexport.exp

```diff
@@ -0,0 +1,33 @@
+__FE_DFL_DISABLE_SSE_DENORMS_ENV
+__FE_DFL_ENV
+___inf
+_exp2
+_exp2$fenv_access_off
+_fdim
+_fdim$fenv_access_off
+_feclearexcept
+_fegetenv
+_fegetexcept
+_fegetround
+_feholdexcept
+_feraiseexcept
+_fesetenv
+_fesetexcept
+_fesetround
+_fetestexcept
+_feupdateenv
+_fmax
+_fmax$fenv_access_off
+_fmin
+_fmin$fenv_access_off
+_log2
+_log2$fenv_access_off
+_modff
+_modff$fenv_access_off
+_nan
+_nanf
+_nextafterd
+_nextafterf
+_remquo
+_remquo$fenv_access_off
+dyld_stub_binder
```

Comment:
`dyld_stub_binder` isn't supposed to be re-exported; it's automatically added by `ld` for each library.
```suggestion
```

Reply:
Done. Forced pushed the change.

Reply:
It's a bit of an oversight when copy-pasting stuff.